src/features/validation/schemaValidation/schemaValidationUtils.ts

```diff
@@ -5,6 +5,9 @@ import addAdditionalFormats from 'ajv-formats-draft2019';
 import type { ErrorObject, Options } from 'ajv';
 import type { JSONSchema7 } from 'json-schema';
 
+import { ValidationMask } from '..';
+import type { ValidationCategory } from '..';
```

Comment:
Strange! Our eslint rules should rewrite these to absolute paths, but that apparently doesn't work with `..` :shrug: Whatever, I don't mind.